lib/flyover_comments/authorization.rb

```diff
@@ -11,6 +11,16 @@ def can_delete_flyover_comment?(comment, user)
       end
     end
 
+    def can_view_flyover_comment?(comment, user)
```

Comment:
-Let's make the pundit call go to `policy.show?`.
-2nd option shouldn't be flag. Let's make that `can_view_flyover_comment?`
-Let's make the else case just true by default (ie. default we assume anyone can view a comment).